kolibri/plugins/device_management/assets/src/views/manage-content-page/channel-list-item.vue

```diff
@@ -183,6 +171,18 @@
           forExport: this.$route.query.for_export,
         });
       },
+      verticalPadding() {
+        return {
+          paddingBottom: `${this.windowGutter}px`,
+          paddingTop: `${this.windowGutter}px`,
+        };
+      },
+      componentTemplate() {
+        if (this.windowIsLarge) {
+          return channelListItemLarge;
+        }
+        return channelListItemSmall;
+      },
```

Comment:
Here's a more complex example that uses two different templates with slots to implement different layouts